facenet/fr_utils.py

```diff
@@ -0,0 +1,201 @@
+#### PART OF THIS CODE IS USING CODE FROM VICTOR SY WANG: https://github.com/iwantooxxoox/Keras-OpenFace/blob/master/utils.py ####
+
+import tensorflow as tf
+import numpy as np
+import os
+import cv2
+from numpy import genfromtxt
+from keras.layers import Conv2D, ZeroPadding2D, Activation, Input, concatenate
+from keras.models import Model
+from keras.layers.normalization import BatchNormalization
+from keras.layers.pooling import MaxPooling2D, AveragePooling2D
+import h5py
+import matplotlib.pyplot as plt
+
+
+_FLOATX = 'float32'
+
+def variable(value, dtype=_FLOATX, name=None):
+    v = tf.Variable(np.asarray(value, dtype=dtype), name=name)
+    _get_session().run(v.initializer)
```

Comment:
Is this code copied from somewhere else? This method is being called but it's not defined `_get_session`

Reply:
is the code in this file working?

Reply:
I have put all the description can you plz check it from file heading